distribution/test/itests/test-itests-alliance/src/test/java/alliance/test/itests/TestNsili.java

```diff
@@ -172,9 +200,9 @@ public void afterAllianceTest() throws Exception {
     }
 
     private void startMockResources() throws Exception {
-        httpWebPort = new DynamicPort("org.codice.alliance.corba_web_port", 6);
-        ftpWebPort = new DynamicPort("org.codice.alliance.corba_ftp_web_port", 7);
-        corbaPort = new DynamicPort("org.codice.alliance.corba_port", 8);
+        httpWebPort = new DynamicPort("org.codice.alliance.corba_web_port", 7);
```

Comment:
I know these were here before this PR, but is there any reason not to have these created statically outside the method?